app/assets/javascripts/pages/agent-edit-page.js.coffee

```diff
@@ -108,10 +108,12 @@ class @AgentEditPage
     $(".control-link-region").show()
 
   hideEventCreation: ->
-    $(".event-related-region").hide()
```

Comment:
In the new Agent form, when I filled it out for an agent type that can create events, then switched it to one that cannot and submitted it, I still saw `receiver_ids` get sent.  I wonder if this is an existing issue?